src/controllers/SelectYearController.ts

```diff
@@ -0,0 +1,150 @@
+import Joi from '@hapi/joi';
+import { Session, SessionMiddleware } from 'ch-node-session-handler';
+import { PenaltyList } from 'ch-sdk-node/dist/services/lfp';
+import { Request, Response } from 'express';
+import { UNPROCESSABLE_ENTITY } from 'http-status-codes';
+import { provide } from 'inversify-binding-decorators';
+import { controller } from 'inversify-express-utils';
+import { BaseController, FormActionHandler, FormActionHandlerConstructor } from './BaseController';
+import { SafeNavigationBaseController } from './SafeNavigationBaseController';
+import { FormActionProcessor } from './processors/FormActionProcessor';
+import { FormValidator } from './validators/FormValidator';
+
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
+import { loggerInstance } from 'app/middleware/Logger';
+import { PenaltyReferenceRouter } from 'app/middleware/PenaltyReferenceRouter';
+import { Appeal } from 'app/models/Appeal';
+import { ApplicationData, APPLICATION_DATA_KEY } from 'app/models/ApplicationData';
+import { createPenaltyRadioButton } from 'app/models/components/PenaltyRadioButton';
+import { createSchema } from 'app/models/fields/PenaltyChoice.schema';
+import { APPLICATION_DATA_UNDEFINED, SESSION_NOT_FOUND_ERROR } from 'app/utils/CommonErrors';
+import { PENALTY_DETAILS_PAGE_URI, REVIEW_PENALTY_PAGE_URI, SELECT_YEAR_PAGE_URI } from 'app/utils/Paths';
+import { ValidationResult } from 'app/utils/validation/ValidationResult';
+
+const template = 'select-the-year';
+
+const navigation = {
+    previous(): string {
+        return PENALTY_DETAILS_PAGE_URI;
+    },
+    next(): string {
+        return REVIEW_PENALTY_PAGE_URI;
+    },
+    actions: (changeMode: boolean) => {
```

Comment:
Currently there is no option to change the penalty details from the summary screen, is this needed?

Reply:
![image](https://user-images.githubusercontent.com/54070318/84754700-a7a9fc80-afb8-11ea-87a5-aa64b541f66d.png)

No you're right however, I used the actions field for consistency as it is used in other action yielding controllers. I can make it indifferent to the change mode